### ISSUE TITLE
NullRef exception when window has never been maximized

### ISSUE THREAD

**Issue by BrainCrumbz** [raflop/ToastNotifications#14]
This is close to issue #13 . In that case, the "main content window" had been maximized and then minimized at least once previously, before actually trying to show a toast.

In this case instead, main content windows has never been maximized since application start. When program logic decides a toast should be generated, another (different) NullReferenceException is thrown. Again, the toast seems to be created because Windows 10 shows its system notification with toast message. The exception is actually thrown a couple of seconds right after that.

From a debug version of DLL, we got [this is](https://github.com/raflop/ToastNotifications/blob/a94e3d8dd328ee621ffcc4293a27da914e997cad/ToastNotifications/NotificationsSource.cs#L94) the offending line, nr 94 in NotificationsSource.cs:

```
n.InvokeHideAnimation();
```

where `InvokeHideAnimation` Action field is null.

**Comment by raflop** [comment]
Hi, same situation as with #13, I couldn't reproduce error, so I made a null checks in NotificationSource in line 94 as you suggested.  Fix is avalaible in 1.4.1 if it's not working for you please provide sample code to reproduce this error.

**Comment by BrainCrumbz** [comment]
I'll let you know as soon as we update and try again. Tnx!

**Comment by BrainCrumbz** [comment]
Just a note on commit addressing these issues, the current one in particular:

[At this line](https://github.com/raflop/ToastNotifications/blob/3c40f5704f89aaef3b57a512358cb79eae9c1b43/ToastNotifications/NotificationsSource.cs#L97):

```
n.InvokeHideAnimation();
```

the exception is not due to `n` being null, but due to its `Action` field `InvokeHideAnimation` being null.

So newly added check:

```
if (n == null)
    return;
```

should not help. I guess it should be something like:

```
if (n.InvokeHideAnimation == null)
    return;
```

**Comment by raflop** [comment]
Hi thanks I will fix this in 1.4.3

**Comment by BrainCrumbz** [comment]
Confirm: 1.4.3 fixes this
